cg/server/schemas/order.py

```diff
@@ -0,0 +1,10 @@
+from typing import List, Optional
```

Comment:
Could this be better placed in cg.models?

Reply:
Sure, I'm struggling with where to put those and similar but you are probably right

Reply:
Changed